lib/ramble/ramble/util/path.py

```diff
@@ -103,8 +103,23 @@ def canonicalize_path(path):
 
 
 def get_maybe_local_path(path):
-    """Convert a scheme-less path to absolute local path"""
-    parsed = urllib.parse.urlparse(path)
+    """Convert a scheme-less path to absolute local path
+
+    Also, remove trailing back-slashes from the input path
```

Comment:
It may be a bit confusing given that the function has "local" in it, but the "removing trailing" applies to any path. Maybe rename the helper to something along the lines of "normalize_path"?

May also be worthwhile to add a test case into the existing test of this helper.

Reply:
Updated. LMK if you think it should be changed anymore.